File: csharp/zebra-puzzle/ZebraPuzzleTests.cs
using Xunit;

public class ZebraPuzzleTests
{
    [Fact]
    public void Resident_who_drinks_water()
    {
        Assert.Equal(Nationality.Norwegian, ZebraPuzzle.DrinksWater());
    }

    [Fact]
    public void Resident_who_owns_zebra()
    {
        Assert.Equal(Nationality.Japanese, ZebraPuzzle.OwnsZebra());
    }
}

File: fsharp/zebra-puzzle/ZebraPuzzleTests.fs
module ZebraPuzzleTests

open FsUnit.Xunit
open Xunit

open ZebraPuzzle

[<Fact>]
let ``Resident who drinks water`` () =
    drinksWater |> should equal Norwegian

[<Fact>]
let ``Resident who owns zebra`` () =
    ownsZebra |> should equal Japanese

